packages/driver/test/cypress/integration/cypress/runner_spec.js

```diff
@@ -32,4 +41,79 @@ describe('src/cypress/runner', () => {
       expect(pending[1].title).to.eq('is pending 2')
     })
   })
+
+  context('async timeouts', () => {
```

Comment:
Move all of the tests from `runner_spec.js` in this suits to an e2e test and actually evaluate the exit code. We want to try and avoid 'modifying reality' by pulling out the error then returning false if we can.

Reply:
kept these, but added e2e test. LMK if i should remove